pages/templates/_slug.vue

```diff
@@ -189,9 +189,11 @@ export default {
     },
     openPreviewModal() {
       this.showPreviewModal = true
+      document.body.style.overflow = "hidden"; //to hide page scrollbar
     },
     closePreviewModal() {
       this.showPreviewModal = false
+      document.body.style.overflow = "auto"; //to add page scrollbar
```

Comment:
What happens if user didn't click close button instead clicked back button from the browser?

Reply:
Right, It will not add scrollbar again.